src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerdeletionservice/config/DataComplianceProperties.kt

```diff
@@ -9,16 +9,20 @@ import org.springframework.validation.annotation.Validated
 @Configuration
 class DataComplianceProperties(
   @Value("\${data.compliance.deletion.enabled:false}") deletionEnabled: Boolean,
-  @Value("\${data.compliance.deceased.deletion.enabled:false}") deceasedDeletionEnabled: Boolean
+  @Value("\${data.compliance.deceased.deletion.enabled:false}") deceasedDeletionEnabled: Boolean,
+  @Value("\${data.compliance.offender.no.booking.deletion.enabled:false}") offenderNoBookingEnabled: Boolean
 ) {
   val deletionEnabled: Boolean
   val deceasedDeletionEnabled: Boolean
+  val offenderNoBookingDeletionEnabled: Boolean
 
   init {
     log.info("Data compliance deletion enabled: {}", deletionEnabled)
     log.info("Data compliance deceased deletion enabled: {}", deceasedDeletionEnabled)
+    log.info("Data compliance offender no booking deletion enabled: {}", offenderNoBookingEnabled)
     this.deletionEnabled = deletionEnabled
     this.deceasedDeletionEnabled = deceasedDeletionEnabled
+    this.offenderNoBookingDeletionEnabled = offenderNoBookingEnabled
```

Comment:
this looks a pointless assignment - why not make the constructor field a `val` instead and then you don't need to copy it over